crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -160,30 +192,44 @@ pub(crate) fn unused_import(checker: &Checker, scope: &Scope, diagnostics: &mut
     }
 
     let in_init = checker.path().ends_with("__init__.py");
-    let fix_init = !checker.settings.ignore_init_module_imports;
+    // TODO: find the `__all__` node
+    let dunder_all = None;
 
-    // Generate a diagnostic for every import, but share a fix across all imports within the same
+    // Generate a diagnostic for every import, but share fixes across all imports within the same
     // statement (excluding those that are ignored).
     for ((node_id, exceptions), imports) in unused {
         let in_except_handler =
             exceptions.intersects(Exceptions::MODULE_NOT_FOUND_ERROR | Exceptions::IMPORT_ERROR);
         let multiple = imports.len() > 1;
+        // FIXME: rename "imports" to "bindings"
+        // FIXME: rename "node_id" to "import_statement"
```

Comment:
reviewer, please approve or deny these renames? i think they'd improve the readability of the code